frontend/website/docs/faq.md

```diff
@@ -66,7 +66,27 @@ It very likely will. However, it is far less of an issue than it was a few years
 
 Coda does not support smart contracts currently. The development team is looking into smart contract models, and it is on the roadmap for future development.
 
-#### Does this protocol offer secret transactions, like in Zcash?
+#### Does Coda support shielded transactions, like in Zcash?
 
 No, Coda does not natively implement privacy features at the moment. However, privacy is key consideration for cryptocurrencies, and is also on the development roadmap.
 
+## SNARKs and Snark Workers
+
+#### If I run a Snark Worker, how do I get paid for my SNARKs that I generate?
+
+Block producers (the validators who add new blocks to the blockchain) are required to buy SNARKs from the network (or from what we call the Snarketplace) and will pay out some of their block reward as fees to the snark workers who generated SNARKs. This creates a secondary incentive mechanism in the network to reward nodes that help compress transactions.
+
+#### Are Snark Workers like Proof-of-Work (PoW) miners?
+
+The only way in which Snark workers are similar to Pow miners is that both snark proving and mining (generating hashes) are parallelizable tasks, meaning that they are optimized for GPUs. However, beyond that, producing SNARKs is not similar to PoW mining. Given the same hardware and proving algorithm, producing Snark proofs always consumes a fixed amount of resources, regardless of how many other nodes are generating SNARKs. This is unlike PoW mining algorithms where the target difficulty increases as more participants join, and the cost of mining increases as a result. In addition, SNARK research is advancing quickly, and we anticipate that SNARK proving will only become cheaper over time as hardware and algorithms get more efficient.
+
+#### Will Snark Workers require more storage and computing power over time? What about compared to Coda full nodes?
+
+Snark workers will not need more storage or computing power over time. Snark workers simply query the mempool for pending transactions requiring Snark proofs, and then generate said proof -- this does not require syncing historical data. In addition, the underlying proving cost of Snark work doesn’t get more expensive with time.
```

Comment:
technically they query the Snark work mempool for pending work (which may be a transaction annotated with metadata or work to merge two existing proofs). It is true it does not require historical data and that it won't need more storage or computing power over time (for a given set of protocol constants, if we change some constants with a hard fork it may change storage/computing power requirements at that moment)